File: JAVA/class object/fact.java
import java.util.Scanner;
class Ada {
    int n;
}
class fact{
    public static void main(String[]args){
     Scanner obj=new Scanner(System.in);
     System.out.println("Enter the value of n=");
     int fac=1,i;
     Ada c=new Ada();
     c.n=obj.nextInt();
     for(i=1;i<=c.n;i++){
        fac=fac*i;
     }   
     System.out.println(fac);
    }
}

File: c++/Function/fact.cpp
#include<iostream>
using namespace std;
int fac();
int main(){
fac();
return 0;
}
int fac(){
    int n,fact=1,i;
    cout<<"Enter the value of n=";
    cin>>n;
    for(i=0;i<=n;i++){
    fact=fact*n;
    i--;
    }
    cout<<fact;
    
    return 0;
}